app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
 end
+module ApplicationHelper
+  def full_title page_title = ""
+      base_title = "Ruby on Rails Tutorial Sample App"
+  page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
indent

Reply:
wrong indent

Reply:
I have fixed it in sublime but did not come out here?